ffbs-parker-nodeconfig-respondd/Makefile

```diff
@@ -0,0 +1,20 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ffbs-parker-nodeconfig-respondd
+PKG_VERSION:=1
+
+PKG_MAINTAINER:=Chris Fiege <chris@tinyhost.de>
+PKG_LICENSE:=MIT
+
+include $(TOPDIR)/../package/gluon.mk
+
+define Package/ffbs-parker-nodeconfig-respondd
+  TITLE:=respondd module for nexthop and gateway info on parker gateway nodes
+  DEPENDS:=+ffbs-parker-nodeconfig +respondd +kmod-nlmon
```

Comment:
As https://github.com/freifunk-gluon/community-packages/pull/142 is still WIP, is this name fixed?

Reply:
Fixed the dependency name for now. But I leave this conversation open until #142 is merged.